atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/memory/InMemoryAtlasDbFactory.java

```diff
@@ -67,6 +69,9 @@
 @AutoService(AtlasDbFactory.class)
 public class InMemoryAtlasDbFactory implements AtlasDbFactory {
 
+    // Only used in deprecated method. See note below.
```

Comment:
Optional: we could make this constant `@Deprecated` too, to make it even more explicit.